pkg/util/azureclient/applens/applens.go

```diff
@@ -25,8 +26,8 @@ type appLensClient struct {
 var _ AppLensClient = &appLensClient{}
 
 // NewAppLensClient returns a new AppLensClient
-func NewAppLensClient(env *azureclient.AROEnvironment, cred azcore.TokenCredential) AppLensClient {
-	client, _ := NewClient(env.AppLensEndpoint, env.AppLensScope, cred, nil)
+func NewAppLensClient(env *azureclient.AROEnvironment, cred azcore.TokenCredential, log *logrus.Entry) AppLensClient {
+	client, _ := NewClient(log, env.AppLensEndpoint, env.PkiIssuerUrlTemplate, env.PkiCaName, env.AppLensScope, cred, nil)
```

Comment:
Does `NewClient` return an error as the second value? If so, why are we not checking that?